work-item-comments.go

```diff
@@ -33,22 +32,18 @@ func (c *WorkItemCommentsController) Create(ctx *app.CreateWorkItemCommentsConte
 			return jsonapi.JSONErrorResponse(ctx, goa.ErrNotFound(err.Error()))
 		}
 
-		currentUser, err := login.ContextIdentity(ctx)
+		currentUserID, err := login.ContextIdentity(ctx)
```

Comment:
The documentation for `login.ContextIdentity` says that it is returning an identity's ID. But here you are assigning the result to `...UserID`. Can *user* and *identity* be used interchangeably?